plugins/JiraSearch-BD32A62C-6F98-4541-AE8E-17B46458595F.json

```
@@ -0,0 +1,12 @@
+{
+  "ID": "BD32A62C-6F98-4541-AE8E-17B46458595F",
+  "Name": "JiraSearch",
```

The name here does not match the name in your plugin's `plugin.json` file.
```suggestion
  "Name": "Jira Search",
```

---

Thank you very much for the very quick review. I really appreciate it. 

I have implemented all the changes in my repo and here. So: Back to you.